miopengemm/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@
 add_definitions(-std=c++11)
 
 set( miopengemm_INSTALL_DIR miopengemm)
+set( lib_SOVERSION 1.1 )
```

Comment:
This variable is passed to the new rocm_set_soversion() function in rocm cmake. Since the MIOpenGEMM didnt have any version, we have added this number(1.1). Please let me know if updates are required.